### ISSUE TITLE
Error getting LMStudio models: fetch failed

### ISSUE THREAD

**Issue by valinux** [stackblitz-labs/bolt.diy#949]
### Describe the bug

Error getting LMStudio models: fetch failed

### Link to the Bolt URL that caused the error

Error getting LMStudio models: fetch failed Error getting LMStudio models: fetch failed Error getting LMStudio models: fetch failed Error getting LMStudio models: fetch failed

### Steps to reproduce

Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed

### Expected behavior

Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed

### Screen Recording / Screenshot

Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed

### Platform

- OS:  Windows

### Additional context

Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed
Error getting LMStudio models: fetch failed

**Comment by leggin2020** [comment]
Yes, I just did a 'git pull' today and I'm receiving the same error.   Please update with a fix.  Thanks!!!!

**Comment by Dev-Adventure** [comment]
Enable CORS in LMStudio
I use `LMSTUDIO_API_BASE_URL=http://127.0.0.1:1234` in `.env` file instead `http://localhost:1234``

![image](https://github.com/user-attachments/assets/9320d789-d43a-4eaa-92f5-a1c1747d8b30)

**Comment by tasktuner** [comment]
same issue, dont have lmstudio installed

**Comment by agzpaoyi123** [comment]
> Enable CORS in LMStudio I use `LMSTUDIO_API_BASE_URL=http://127.0.0.1:1234` in `.env` file instead `http://localhost:1234`
> 
> ![image](https://private-user-images.githubusercontent.com/84775314/399421676-9320d789-d43a-4eaa-92f5-a1c1747d8b30.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU3MjA4MDAsIm5iZiI6MTczNTcyMDUwMCwicGF0aCI6Ii84NDc3NTMxNC8zOTk0MjE2NzYtOTMyMGQ3ODktZDQzYS00ZWFhLTkyZjUtYTFjMTc0N2Q4YjMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTAxVDA4MzUwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjZjUxMDYzZjBkODNiNWJjZjcxOWE3NzgwNjQ2OGQ2YTJmZmEzMjc0YTU2ZDBkNmU1M2Y2NDU4ODY4YmUxNWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.gF5xP3E0I_nwuG9Ykco5JTWvgZ1_jne1uuasd_XJCzU)

It works.

**Comment by leex279** [comment]
See last comment from @agzpaoyi123
